### PR TITLE
Update PR template

### PR DESCRIPTION
**Pull request by dpryan79**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [x] This PR does something else (explain below).

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
 
 * [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
 * [ ] This PR adds a new recipe.
 * [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
 * [ ] This PR updates an existing recipe.
 * [ ] This PR does something else (explain below).
+
+Everyone has access to the following BiocondaBot commands, which can be given in a comment:
+
+ * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
```
Comment (mbargull) [inline .github/PULL_REQUEST_TEMPLATE.md]: Does the bot know not react to `@BiocondaBot command` if it's part of a code block?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
+ * `@BiocondaBot please add label` will add the `please review & merge` label.
+ * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
+
+For members of the Bioconda project, the following command is also available:
+
+ * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
+
+If you have questions, please post them in gitter or ping `@bioconda/core` in a comment.
```
Comment (mbargull) [inline .github/PULL_REQUEST_TEMPLATE.md]: Maybe mention that @BiocondaBot will take over pinging duty for non-members?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (dpryan79) [reply .github/PULL_REQUEST_TEMPLATE.md]: Nope, a comment has to start with `@BiocondaBot` or `@bioconda-bot`

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (dpryan79) [reply .github/PULL_REQUEST_TEMPLATE.md]: Good idea.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (mbargull) [reply .github/PULL_REQUEST_TEMPLATE.md]: Perfect! So it wouldn't go berserk on every repost of this template; that's good ;).

Comment (dpryan79) [reply .github/PULL_REQUEST_TEMPLATE.md]: Thankfully not :)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (dpryan79) [reply .github/PULL_REQUEST_TEMPLATE.md]: I've mentioned that now, though I'm not sure how clear that is.

Comment (mbargull) [reply .github/PULL_REQUEST_TEMPLATE.md]: Looks good. If it's someone mentions that it's unclear, we can amend that later to be more specific ("even non-Bioconda members can ping" or something). I think it's fine as is.

**Pull request merged**